nullaway/src/main/java/com/uber/nullaway/LibraryModels.java

```diff
@@ -112,6 +112,15 @@ public interface LibraryModels {
    */
   ImmutableSet<MethodRef> nonNullReturns();
 
+  /**
+   * Get the (className, generic type argument) pairs for library Classes where the generic type
+   * argument has a Nullable upper bound.
+   *
+   * @return map from the className to the position of the generic type argument that has a Nullable
+   *     upper bound.
+   */
+  ImmutableSetMultimap<String, Integer> typeVariablesWithNullableUpperBounds();
```

Comment:
So should we also extend the new Empty library models abstract class for the classes currently implementing LibraryModels interface. for e.g the Example Library Models? So that the default implementations are kept in the EmptyLibraryModels.

Reply:
Yeah, that's what I'm thinking.  Our current implementations should extend the new empty library models class.

Reply:
Thanks. Makes sense, I'll do that.

Reply:
Ok, really sorry here, but I realized we are going about this the wrong way, and we really should be using the language feature introduced for just this case, [default methods in interfaces](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html).  There is no need for introducing the `EmptyLibraryModels` class; we should just make `typeVariablesWithNullableUpperBounds()` a `default` method that returns the empty set and then all existing implementations will just work without any changes.

@akulk022 could you do the following:
1. Make `typeVariablesWithNullableUpperBounds()` a `default` method.
2. Also make `LibraryModels.nullableFields()` a `default` method that returns an empty set.  We should have done this when introducing this method before but I forgot.
3. Get rid of `EmptyLibraryModels`, and undo the changes to `ExampleLibraryModels` and `TestLibraryModels`.

Sorry again for not remembering about `default` methods but they are definitely the best way to go here.

Reply:
No problem, It's a nice language feature to deal with such a problem and I should have known it exists, I will certainly remember it now.